pkg/service/v0/service.go

```diff
@@ -99,22 +103,34 @@ func (o Ocs) GetUser(w http.ResponseWriter, r *http.Request) {
 func (o Ocs) GetSigningKey(w http.ResponseWriter, r *http.Request) {
 
 	// TODO move token marshaling to ocis-proxy
```

Comment:
is this TODO still relevant? if so, worth transforming it into an issue?

Reply:
yep, tracking in https://github.com/owncloud/ocis-ocs/issues/19 and removing the todo in the code